cetmix_tower_server/models/cx_tower_variable_value.py

```diff
@@ -57,6 +57,8 @@ class TowerVariableValue(models.Model):
         compute="_compute_variable_ids",
         store=True,
     )
+    required = fields.Boolean()
+
```

Comment:
"Required" means that this value should be used when creating a new server. At the same time this value can remain empty in the server template. Because it will be populated later in the server creation wizard.